src/si/heat_transfer.rs

```diff
@@ -0,0 +1,329 @@
+//! Quantity of thermodynamic heat transfer, also known as heat transfer coefficient,
```

Comment:
The quantity name "thermodynamic heat transfer" doesn't make sense to me, and I'm having a very hard time finding anything that uses such a name to refer to the concept. I would prefer to see this renamed to "film effectiveness" personally (largely because having "coefficient" as a quantity doesn't seem very rigorous to me, so adding "coefficient" feels like a hack).

Reply:
Hello @Aehmlo, thanks for the feedback.

"Heat Transfer" was the most common name in the engineering literature we were referring to while using this quantity. "Film Effectiveness" is called out here as a common alias here in the interest of discoverability. Perhaps "Film Effectiveness" is a better name in some context we were not engaged with? We can certainly change it to "Flim Effectiveness" if it is more clear, discoverable, or correct.

Some academic sources:
Mejia Parra, Daniel & Montoya-Zapata, Diego & Arbelaiz, Ander & Moreno, Aitor & Posada, Jorge & Ruiz, Oscar. (2018). Fast Analytic Simulation for Multi-Laser Heating of Sheet Metal in GPU. Materials. 11. 2078. 10.3390/ma11112078.

Maria, Rutheravan. (2016). HEAT TRANSFER COEFFICIENT OF NANOFLUID. 10.13140/RG.2.1.4647.4642. 

Some engineering literature:
https://books.google.com/books?id=u4U1H13ksmQC&printsec=frontcover#v=onepage&q&f=false

Some internet citations:
http://www.thermopedia.com/content/841/
https://en.wikipedia.org/wiki/Heat_transfer_coefficient
https://www.engineersedge.com/thermodynamics/overall_heat_transfer-table.htm

Reply:
[NIST 811](https://www.nist.gov/pml/nist-guide-si-appendix-b9-factors-units-listed-kind-quantity-or-field-science) uses "coefficient of heat transfer." I'm inclined towards `HeatTransferCoefficient` currently.

@Aehmlo can you expand on why including coefficient doesn't seem as rigorous to you?

Reply:
We'd leaned away from `HeatTransferCoefficient` in favor of `HeatTransfer` because all of these quantities could be considered coefficients when constructed with values. As such, putting `Coeffiicient` on one _quantity_ seemed strange, but its a minor change if desired.